test/e2e/gitops_service_test.go

```diff
@@ -237,3 +245,34 @@ func tearDownArgoCD(t *testing.T) {
 	assertNoError(t, err)
 
 }
+
+func deployClusterScopedManifest(t *testing.T) {
```

Comment:
can we rename this to something more related to what we are testing ? probably something like validateOpenShiftImageDeployment or validateMachineConfigUpdates ?

Reply:
Done.
```validateMachineConfigUpdates```